client/state/data-layer/wpcom/activity-log/delete-credentials/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { JETPACK_CREDENTIALS_DELETE, JETPACK_CREDENTIALS_STORE } from 'state/action-types';
+import { successNotice, errorNotice } from 'state/notices/actions';
+
+export const request = ( { dispatch }, action ) => {
+	const notice = successNotice( i18n.translate( 'Deleting credentials…' ), { duration: 4000 } );
```

Comment:
Do we need to issue these notices on this? In general I haven't seen us follow this idiom around Calypso. Maybe a designer could provide some helpful advice here.